resources/lang/en/admin/hardware/message.php

```diff
@@ -7,6 +7,7 @@
     'does_not_exist' 	=> 'Asset does not exist.',
     'does_not_exist_or_not_requestable' => 'Nice try. That asset does not exist or is not requestable.',
     'assoc_users'	 	=> 'This asset is currently checked out to a user and cannot be deleted. Please check the asset in first, and then try deleting again. ',
+    'no_assets_selected' => 'You must select at least one asset from the list',
```

Comment:
Refactoring this made the most sense to me. But let me know if there is a different way.